perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1281,6 +1281,11 @@ sub user_edit : Path('user_edit') : Args(1) {
             $user->from_body( undef );
         }
 
+        # Some cobrands (e.g. Oxfordshire) have their own custom fields on User
```

Comment:
Does the approach I've taken in the defect types branch ([Admin/DefectTypes.pm](https://github.com/mysociety/fixmystreet/blob/137510fe0a74226b10fe884498a03bb7c5b3b9be/perllib/FixMyStreet/App/Controller/Admin/DefectTypes.pm#L74-L76) and [Oxfordshire.pm](https://github.com/mysociety/fixmystreet/blob/137510fe0a74226b10fe884498a03bb7c5b3b9be/perllib/FixMyStreet/Cobrand/Oxfordshire.pm#L180-L185)) make sense for this?